packages/components/src/input-control/utils.js

```diff
@@ -3,33 +3,27 @@
  */
 import { useEffect } from '@wordpress/element';
 
-/**
- * Internal dependencies
- */
-import { getRTL } from '../utils/style-mixins';
-
 /**
  * Gets a CSS cursor value based on a drag direction.
  *
  * @param {string} dragDirection The drag direction.
  * @return {string} The CSS cursor value.
  */
 export function getDragCursor( dragDirection ) {
-	const isRtl = getRTL();
-	let dragCursor = 'n-resize';
+	let dragCursor = 'ns-resize';
 
 	switch ( dragDirection ) {
 		case 'n':
-			dragCursor = 'n-resize';
+			dragCursor = 'ns-resize';
```

Comment:
Interaction improvements. Favouring `ns` / `ew` resize cursors vs. single coordinate cursors. The multi-coordinate cursors work better as the visually indicate the directions a user can drag.

Reply:
does it make sense that we group the cases like:

```
case 'n':
case 's':
    dragCursor: 'ns-resize';
    break;

case 'e':
case 'w':
    dragCursor: 'ew-resize';
    break;
```